### PR TITLE
Rewrite queries

### PR DESCRIPTION
**Pull request by ankitgit**
Added dep to supply settings via command line

### CODE REVIEW

File: src/cmd_tool.rs [hunk 1]
```diff
@@ -6,102 +6,119 @@ use simulation_curator::gtfs::ShapePoint;
...
+struct Args {
+    /// Path to the gtfs database
+    #[arg(short, long, default_value = "gtfs_vbb.db")]
+    dbPath: String,
+
+    ///  The time of the day from when the schedule needs to be selected
+    #[arg(short, long, default_value = "08:00:00")]
+    start_time: String,
+
+    /// The time of the day till when the schedule needs to be selected
```
Comment (fkuzun) [inline src/cmd_tool.rs]: should be "until when"

File: src/cmd_tool.rs [hunk 2]
```diff
@@ -6,102 +6,119 @@ use simulation_curator::gtfs::ShapePoint;
...
+    #[arg(short, long, default_value = "09:00:00")]
+    end_time: String,
+
+    /// The day of the week for which the schedule needs to be selected
+    #[arg(short, long, default_value = "monday")]
+    day: String,
+
+    /// The time interval in seconds that need to be represented by one second. This parameter allows us to speedup the time to increase the rate of topology changes.
```
Comment (fkuzun) [inline src/cmd_tool.rs]: This does not in all cases represent one second. Only if the change frequency is set to 1

File: src/cmd_tool.rs [hunk 3]
```diff
@@ -6,102 +6,119 @@ use simulation_curator::gtfs::ShapePoint;
...
+    #[arg(short, long, default_value_t = 20)]
+    batch_interval_size_in_seconds: u64,
+
+    /// The frequency at which the topology changes need to be produced
```
Comment (fkuzun) [inline src/cmd_tool.rs]: The variable actually describes the frequency of batches. We should therefore also adjust the name for the parameter to batch_frequency

File: src/cmd_tool.rs [hunk 4]
```diff
@@ -6,102 +6,119 @@ use simulation_curator::gtfs::ShapePoint;
...
+
+    /// Path to the file where geo.json will be produced
+    #[arg(short, long, default_value = "geo.json")]
+    geo_json_path: String,
+
+    /// Name of the csv file containing OpenCelliD data.
+    #[arg(short, long, default_value = "OpenCelliDGermanyData.csv")]
+    open_cell_id_data_loc: String,
+
+    /// The number of people connected to the base station.
```
Comment (fkuzun) [inline src/cmd_tool.rs]: "The minimum number of measurements required for a cellular base station to be included in the experiment"

File: src/cmd_tool.rs [hunk 5]
```diff
@@ -6,102 +6,119 @@ use simulation_curator::gtfs::ShapePoint;
...
+    #[arg(short, long, default_value_t = 10)]
+    min_samples: u64,
+
+    /// The type of network the base station support. We use LTE only for our experiments.
```
Comment (fkuzun) [inline src/cmd_tool.rs]: supportS

File: src/cmd_tool.rs [hunk 6]
```diff
@@ -6,102 +6,119 @@ use simulation_curator::gtfs::ShapePoint;
...
-    //     Ok(row.get::<usize, String>(0))
-    // })?;
-
-
+    // get routes and trips for a specific calender date
+    let mut stmt = db.prepare("SELECT DISTINCT block_id \
+                                             FROM routes, trips, calendar_dates \
+                                             WHERE trips.route_id=routes.route_id \
+                                                    AND trips.service_id=calendar_dates.service_id \
+                                                    AND routes.route_short_name in ('S41') \
```
Comment (fkuzun) [inline src/cmd_tool.rs]: Should the line name and date remain hardcoded?

**Review by fkuzun — COMMENTED** [review]
LGTM, some minor changes required

File: src/cmd_tool.rs (hunk 6 above)
Comment (ankitgit) [reply src/cmd_tool.rs]: I am thinking of changing the query to take line name as input param and replace date with day

**Pull request merged**